automation_tools/__init__.py

```diff
@@ -1074,7 +1076,10 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         cdn_version=sat_cdn_version,
         host=host
     )
-
+    # Create custom beaker repo
+    if os.environ.get('OS_UPGRADE_REPO'):
+        beaker_repo_url = os.environ.get('OS_UPGRADE_REPO')
+        execute(create_custom_repos, beaker_server=beaker_repo_url, host=host)
```

Comment:
True, but there is **kwargs argument. So it can be any argument.
" ... Where the key is the repository filename and repository name, and the value is the
    repository URL."